financials/income_log.csv

```diff
@@ -4,4 +4,9 @@ income_id,date,type,source,currency,amount,exchange_rate,USD_equivalent,to_addre
 3,20190303,donation,Sparkpool,GRIN,5500,3.63,20000.29,https://donations.grin-tech.org,817e0a69-5ab0-421f-afc3-406f61dd2bb4,Sparkpool & anonymous miner donation
 4,20190304,donation,BlockCypher,USD,3600,1,3600,Discourse,-,BlockCypher financing last 6 months and next 2 years (so 30 months) of Discourse subscription (120 USD/month)
 5,20190301,donation,Grin-pool.org,GRIN,17.087656585,3.27,55.88,https://donations.grin-tech.org,-,https://hendi.io/february-2019-our-first-donation-to-grin/
-6,20190309,donation,Bminer,BTC,3.2336,4017.3,12990.34,3ChVP627KU5w4zu2rieFPF3wGXWQgmhvrs,ce86a17cfa256efc9a72190aee223ec75a38e03fc86ba2858c7c2d10d6dddfc3,Donation
\ No newline at end of file
+6,20190309,donation,Bminer,BTC,3.2336,4017.3,12990.34,3ChVP627KU5w4zu2rieFPF3wGXWQgmhvrs,ce86a17cfa256efc9a72190aee223ec75a38e03fc86ba2858c7c2d10d6dddfc3,Donation
+7,20180907,donation,Aurel,ETH,183.0,39747.60,0xb010e9160e75a9adb2139deecfdf2c307fd5a0ef,https://etherscan.io/tx/0xff22967cc3c6ca951e23140ff41d273262f6ba22f130ff049555a38345b5dbe7,Yeastplume's 3rd funding campaign: https://www.grin-forum.org/t/funding-campaign-yeastplume-oct-2018-to-feb-2019-closed/677
+8,20181220,donation,Aurel,BTC,6.1169,22890.66,3ChVP627KU5w4zu2rieFPF3wGXWQgmhvrs,https://www.blockchain.com/btc/tx/85251acf39fb801d04bd285ce91c115b875edc29a36103975212043f5e5ff2c4,Security audit funding campaign: https://www.grin-forum.org/t/security-audit-funding-campaign/962
+9,20190201,donation,Qtum,BTC,10.0,34605.50,3B3qq3EAPtpkSoGHSDfY6rxpK5Bqh7zmij,https://www.blockchain.com/btc/tx/85251acf39fb801d04bd285ce91c115b875edc29a36103975212043f5e5ff2c4,Yeastplume's 4th funding campaign: https://www.grin-forum.org/t/funding-campaign-yeastplume-march-to-aug-2019/1697

```

Comment:
@garyyu it looks like the (9) Qtum TXID (https://www.blockchain.com/btc/tx/85251acf39fb801d04bd285ce91c115b875edc29a36103975212043f5e5ff2c4) is actually the same as (8) Aurel - are you able to find the correct tx reference for the 10BTC donation?

Reply:
Oh my!  should be copy/paste issue, will fix it now :-)